client/src/styles/components/_renku_search.scss

```
@@ -121,3 +125,29 @@
     }
   }
 }
+
+.rk-search-result-grid {
+  display: -webkit-box; 
+  display: -ms-flexbox; 
+  display: flex;
+  margin-left: -1em;
+  width: auto;
+
+  .rk-search-result-grid_column {
+    padding-left:1em;
```

For spacing consistency.

```suggestion
    padding-left: 1em;
```